src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/CreateOrganizationOverlay.tsx

```diff
@@ -0,0 +1,128 @@
+// Libraries
+import React, {FC, useCallback, useContext, useState} from 'react'
+import {useDispatch} from 'react-redux'
+import {
+  Button,
+  ComponentColor,
+  ComponentStatus,
+  FlexBox,
+  FlexDirection,
+  Overlay,
+} from '@influxdata/clockface'
+
+// Components
+import {OverlayContext} from 'src/overlays/components/OverlayController'
+import {InputWithLabel} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/InputWithLabel'
+import {ProviderChooser} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ProviderChooser'
+
+// Styles
+import './CreateOrganizationOverlay.scss'
+import {
+  CreateOrgProvider,
+  CreateOrgContext,
+} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/CreateOrganizationContext'
+
+// Types
+import {RemoteDataState} from 'src/types'
+import {notify} from 'src/shared/actions/notifications'
+import {orgCreateSuccess} from 'src/shared/copy/notifications'
+
+const DEFAULT_ERROR_MESSAGE = 'This is a required field'
+const NAME_CONFLICT_MESSAGE =
+  'This organization name already exists. Please choose a different name.'
+
+const CreateOrgOverlay: FC = () => {
+  const {onClose} = useContext(OverlayContext)
+  const {orgName, changeOrgName, createOrg, createOrgLoadingStatus} =
+    useContext(CreateOrgContext)
+  const dispatch = useDispatch()
+  const [errorMessage, setErrorMessage] = useState(DEFAULT_ERROR_MESSAGE)
+  const [showError, setShowError] = useState(false)
+
+  const handleOrgNameChange = useCallback(
+    e => {
+      changeOrgName(e.target.value)
+      setShowError(false)
+      setErrorMessage(DEFAULT_ERROR_MESSAGE)
+    },
+    [changeOrgName]
+  )
+
+  const handleCreateOrg = useCallback(() => {
+    if (orgName.trim() === '') {
+      return
+    }
```

Comment:
Do we want to give feedback to the user instead of just `return`ing if they try to enter an org with no name?

Reply:
updated. should be available in the next commit.

Reply:
Did we resolve this one - is `setErrorMessage` what was added?

Reply:
that's correct. I added
```
setShowError(true)
setErrorMessage(DEFAULT_ERROR_MESSAGE)
```